src/Elastic.Apm.NLog/ApmServiceVersionLayoutRenderer.cs

```diff
@@ -0,0 +1,19 @@
+using System.Text;
+using NLog;
+using NLog.Config;
+using NLog.LayoutRenderers;
+
+namespace Elastic.Apm.NLog;
+
+[LayoutRenderer(Name)]
+[ThreadSafe]
```

Comment:
Please also include `[ThreadAgnostic]`. See also https://github.com/NLog/NLog/wiki/How-to-write-a-custom-layout-renderer#how-to-optimize-performance-for-the-layout-renderer